docs/framework/winforms/controls/add-and-remove-items-with-wf-listview-control-using-the-designer.md

```diff
@@ -18,7 +18,7 @@ The process of adding an item to a Windows Forms <xref:System.Windows.Forms.List
   
 1. Select the <xref:System.Windows.Forms.ListView> control.  
   
-2. In the **Properties** window, click the **Ellipsis** (![VisualStudioEllipsesButton screenshot](../media/vbellipsesbutton.png "vbEllipsesButton")) button next to the <xref:System.Windows.Forms.ListView.Items%2A> property.  
+2.  In the **Properties** window, click the **Ellipsis** (![Use the ellipsis button to access the CellStyle Builder dialog box.](./media/add-and-remove-items-with-wf-listview-control-using-the-designer/visual-studio-ellipsis-button.png)) button next to the <xref:System.Windows.Forms.ListView.Items%2A> property.  
```

Comment:
I'm also thinking since the same image is heavily used in this case, that we should "break" the rules in this case and keep only one copy of that directly under the media folder.